Src/.sarif/README.txt

```diff
@@ -0,0 +1,2 @@
+The solution RE2.Native.sln needs this directory to generate the SARIF files when
+building in debug mode. If the folder does not exist, an exception will occur.
```

Comment:
The changes look solid. I haven't had a chance to check them locally yet, though.

Do we have any documentation already on this type of change that we could link to here, at least? This level of documentation is a little light. I'm okay if that documentation just makes it way into our team process documentation. Happy to take a follow up task to ensure this gets done.